gui/src/app/pyodide/pyodideWorker/pyodideWorkerInterface.ts

```diff
@@ -45,41 +37,37 @@ class PyodideWorkerInterface {
       } else if (msg.type === "stderr") {
         callbacks.onStderr(msg.data);
       } else if (msg.type === "setData") {
-        if (mode !== "data.py") {
-          console.error("setData is only supported in data.py mode");
-          return;
-        }
         if (!callbacks.onData) {
           console.error("onData callback is required for data.py mode");
```

Comment:
May want to revise text to remove reference to modes

Reply:
Thought I had, whoops.